assets/js/phoenix_live_view/rendered.js

```diff
@@ -17,6 +17,66 @@ import {
   isCid,
 } from "./utils"
 
+export let modifyRoot = (html, attrs, innerHTML) => {
+  html = html.trimStart()
+  let tagStartsAt = null
+  let pos = 0
+  while(pos < html.length){
+    let maybeStart = html.indexOf("<", pos)
+    if(maybeStart === -1){ break }
+    if(maybeStart >= 0 && html.charAt(maybeStart + 1) !== "!"){
+      tagStartsAt = maybeStart
+      break
+    }
+    pos += html.indexOf("-->", pos)
+  }
+
+  let commentBefore = tagStartsAt === 0 ? null : html.slice(0, tagStartsAt).trim()
+  let contentAfter = null
+  html = html.slice(tagStartsAt).trimStart()
+  let tagNamesEndsAt
+  for(let i = 1; i < html.length; i++){
+    let char = html.charAt(i)
+    if([">", " ", "\n", "\t", "\r"].indexOf(char) >= 0 || (char === "!" && html.charAt(i + 1) === ">")){
+      tagNamesEndsAt = i
+      break
+    }
+  }
+
+  let tag = html.slice(1, tagNamesEndsAt)
+  let tagOpenEndsAt = html.indexOf(">")
```

Comment:
This does not work because you may have ">" inside an attribute. :D That's why my original suggestion in the gist was to have a list of void tags. Because if you want to check if the tag is actually void, you will get pretty close to having to parse all HTML!